graylog2-plugin-interfaces/src/test/java/org/graylog2/plugin/BaseConfigurationTest.java

```diff
@@ -44,6 +44,9 @@
         @Parameter(value = "rest_listen_uri", required = true)
         private URI restListenUri = URI.create("http://127.0.0.1:12900/");
 
+        @Parameter(value = "web_listen_uri", required = true)
+        private URI webListenUri = URI.create("http://127.0.0.1:9000/");
```

Comment:
ah right, I forgot that one aspect: I would vote for eventually moving the REST API into /rest/* to avoid the clashes.
We can discuss this later though :+1: